HunterPie/GUIControls/Custom Controls/MinimalHealthBar.xaml.cs

```diff
@@ -20,8 +21,9 @@ public double Value
             set
             {
                 valueProperty = value;
-                double v = Math.Max(MaxSize * (value / Math.Max(MaxValue, 1)), 0);
-                HealthBar.Width = v;
+                HealthBar.Width = UserSettings.PlayerConfig.Overlay.MonstersComponent.HideHealthInformation
```

Comment:
This is not a good idea, modifying the control behaviour instead of the widget behaviour would break every single MinimalHealthBar, not only the stamina and health bars.